pkg/datasource/csvsource/csv_tick_converter.go

```diff
@@ -0,0 +1,73 @@
+package csvsource
+
+import (
+	"time"
+
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+var klines []types.KLine
```

Comment:
this is global var, could we avoid this?

Reply:
let me have a look